src/pymortests/algorithms/pod.py

```diff
@@ -28,12 +29,34 @@ def test_pod(vector_array, method):
 
     B = A.copy()
     orth_tol = 1e-10
-    U, s = pod(A, method=method, orth_tol=orth_tol)
+    U, s = pod(A, method=method, orth_tol=orth_tol, return_reduced_coefficients=False)
     assert np.all(almost_equal(A, B))
     assert len(U) == len(s)
     assert np.allclose(U.gramian(), np.eye(len(s)), atol=orth_tol)
 
 
+@settings(deadline=None, suppress_health_check=[HealthCheck.filter_too_much,
+          HealthCheck.too_slow, HealthCheck.data_too_large])
+@given_vector_arrays(method=sampled_from(methods))
+def test_pod_with_coefficients(vector_array, method):
+    A = vector_array
+    # TODO assumption here masks a potential issue with the algorithm
+    #      where it fails in internal lapack instead of a proper error
+    # assumptions also necessitate the health check exemptions
+    assume(len(A) > 1 or A.dim > 1)
+    assume(not contains_zero_vector(A, rtol=1e-13, atol=1e-13))
+
+    B = A.copy()
+    orth_tol = 1e-10
+    U, s, Vh = pod(A, method=method, orth_tol=orth_tol, return_reduced_coefficients=True)
+    assert np.all(almost_equal(A, B))
+    assert len(U) == len(s) == len(Vh)
+    if len(s) > 0:
+        U.scal(s)
+        UsVh = U.lincomb(Vh.T)
+        assert spla.norm((A - UsVh).norm() / spla.norm(A.norm())) < 1e-7
```

Comment:
```suggestion
        assert spla.norm((A - UsVh).norm()) / spla.norm(A.norm()) < 1e-7
```

Slightly more efficient...